docs/source/02-02-sso.md

```diff
@@ -129,3 +125,8 @@ var embed = Coral.createStreamEmbed({
   }
 });
 ```
+## Troubleshooting JWT Validation Errors
+
+Each `user.id`/`user.email` combination must be unique inside Coral. This is true for both Single Sign On users created by JWT tokens, as well as users that register/login using Coral’s built in “Login with email authentication”. You can not share authentication strategies for a single user; thus if a user logs in with SSO, they can not also login with email and vice versa. When you attempt to authenticate a JWT token with an email address that already exists in Coral with a different `user.id` than was passed on the token Coral will throw a Duplicate User error. 
```

Comment:
It’s not just the combination that is unique, it’s each individually.

Reply:
Clarified the constraints better.